src/Nox.Types.EntityFramework/Types/PhoneNumber/PhoneNumberDatabaseConfigurator.cs

```diff
@@ -0,0 +1,29 @@
+﻿
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Nox.Solution;
+using Nox.Types.EntityFramework.Abstractions;
+
+namespace Nox.Types.EntityFramework.Types;
+public class PhoneNumberDatabaseConfigurator : INoxTypeDatabaseConfigurator
+{
+    public NoxType ForNoxType => NoxType.PhoneNumber;
+
+    public virtual bool IsDefault => true;
+
+    public void ConfigureEntityProperty(NoxSolutionCodeGeneratorState noxSolutionCodeGeneratorState, EntityTypeBuilder builder, NoxSimpleTypeDefinition property, Entity entity, bool isKey)
+    {
+        builder
+            .Property(property.Name)
+            .IsRequired(property.IsRequired)
+            .HasMaxLength(30)
```

Comment:
Can we also add a validation in the PhoneNumber type for max length?

Reply:
@dejobratic done